src/app/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import './app.scss';
+import * as mac from 'assets/imgs/mac.png';
+import * as windows from 'assets/imgs/windows.png';
+import * as linux from 'assets/imgs/linux.png';
+import * as Secure from 'assets/imgs/icon-shield.svg';
+import * as Offline from 'assets/imgs/icon-offline.svg';
+import * as Yours from 'assets/imgs/icon-hdd.svg';
+import * as OpenSource from 'assets/imgs/icon-github.svg';
+import { Block } from './block';
+import { Platform } from './platform';
+
+const builds = [
+  { name: 'Mac', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Windows', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Linux', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'HTML Build', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' }
+];
+
+let OS: string = '';
+if (navigator.appVersion.includes('Win')) {
+  OS = 'Windows';
+} else if (navigator.appVersion.includes('Mac')) {
+  OS = 'Mac';
+} else if (navigator.appVersion.includes('Linux')) {
+  OS = 'Linux';
+} else if (navigator.appVersion.includes('X11')) {
+  OS = 'Unix';
+}
+const img: any = OS === 'Mac' ? mac : OS === 'Windows' ? windows : linux;
```

Comment:
Maybe use OSX as the default screenshot in case `OS === ''`. Linux is the ugliest, IMO.

Reply:
I actually tried this, and it didn't work :/

I'll take another look at it and see if I can figure out another solution